xknx/config/config_v1.py

```diff
@@ -94,8 +116,29 @@ def _parse_connection_prefs(self, conn_type: ConnectionType, prefs) -> None:
                     connection_config.gateway_port = value
                 elif pref == "local_ip":
                     connection_config.local_ip = value
+                elif pref == "route_back":
+                    connection_config.route_back = value
         self.xknx.connection_config = connection_config
 
+    def env_connection(self):
+        """Read config from env vars."""
+        gateway_ip = os.getenv("XKNX_CONNECTION_GATEWAY_IP", default=None)
+        if gateway_ip:
+            logger.debug("XKNX_CONNECTION_GATEWAY_IP overwrite from env")
+            self.xknx.connection_config.gateway_ip = gateway_ip
+        gateway_port = os.getenv("XKNX_CONNECTION_GATEWAY_PORT", default=None)
+        if gateway_port:
+            logger.debug("XKNX_CONNECTION_GATEWAY_PORT overwrite from env")
+            self.xknx.connection_config.gateway_port = int(gateway_port)
+        local_ip = os.getenv("XKNX_CONNECTION_LOCAL_IP", default=None)
+        if local_ip:
+            logger.debug("XKNX_CONNECTION_LOCAL_IP overwrite from env")
+            self.xknx.connection_config.local_ip = local_ip
+        route_back = os.getenv("XKNX_CONNECTION_ROUTE_BACK", default=None)
+        if route_back:
+            logger.debug("XKNX_CONNECTION_ROUTE_BACK overwrite from env")
+            self.xknx.connection_config.route_back = bool(route_back)
```

Comment:
I think `XKNX_CONNECTION_ROUTE_BACK = False` as Env will evaluate `true` here!?

Reply:
You are right.